website/src/components/canvas/sidebar/Sidebar.jsx

```diff
@@ -180,6 +219,30 @@ class Sidebar extends React.Component {
             <h3>Resize</h3>
             <TextField hintText="64px" floatingLabelText="Height" style={styles.field} />
             <TextField hintText="64px" floatingLabelText="Width" style={styles.field} />
+
+            <Divider />
+
+            <h3>Rendering</h3>
+            <Checkbox
```

Comment:
Can you change the default color to the same as the text in the buttons (I believe it's grey900 or black). While checked, can you change the color to be red?